content/en/workflows/trigger.md

```diff
@@ -43,6 +43,11 @@ To run the workflow:
 To trigger a workflow from a Monitor:
 1. On the workflow canvas, click **Add an Automated Trigger** and select **@mention**.
 1. Next to **@workflow-**, enter a mention name for the trigger. Your mention name must be unique.
+1. Add a service account:
+   1. Under **Configure**, add a service account next to **@worklow-** to dynamically create a service account and associate it with the workflow.
```

Comment:
It looks like there's a typo (`worklow` instead of `workflow`) here and in a few other places.